src/lib/components/Layers/Layers.js

```diff
@@ -74,7 +74,14 @@ export class Layers extends Component {
             // split to remove .json part
             activeId = activeId.split('.')[0];
           }
-
+          // if (auth.isAuthenticated && (!authConfigs || !authConfigs.LAYERS)) {
```

Comment:
When implementing oauth in gisida at times we have authConfigs not set. This has been observed in sites like cambodia and idai recovery which are public at the moment and currently have Layer.all for a public user i.e access to all layers. The  condition checks if the user is authenticated and if the authconfigs are not set then allows for the layer visibility. This is a hacky fix and the root cause may be found here https://github.com/onaio/gisida/blob/0419a9530d9b168675c76b5fee6dff11d505200a/src/utils/supAuth.js#L115